src/main/java/techreborn/blocks/cable/CableBlock.java

```diff
@@ -136,6 +136,10 @@ private BlockState makeConnections(World world, BlockPos pos) {
 	private Boolean canConnectTo(IWorld world, BlockPos pos, Direction facing) {
 		BlockEntity blockEntity = world.getBlockEntity(pos);
 		if (blockEntity != null && (Energy.valid(blockEntity) || blockEntity instanceof CableBlockEntity)) {
+			if (blockEntity instanceof CableBlockEntity && ((CableBlockEntity)blockEntity).getCableType().tier != type.tier) {
```

Comment:
I thought that different cable types should be able to connect to each others.

Reply:
I quite like this idea, better than explosions, but there needs to be some form of backwards compatibliity.